### PR TITLE
community/rust: upgrade to 1.28.0

### PR DESCRIPTION
**Pull request by jeizsm**
cargo-tests-fix-build-target.patch can be removed if this be merged to cargo https://github.com/rust-lang/cargo/pull/5859

### CODE REVIEW

**Comment by mati865** [comment]
Already commited by @jirutka in the meantime https://github.com/alpinelinux/aports/commit/e6200a5c61a35d55afeef51bd3117ddeef751aec

File: community/rust/APKBUILD [hunk 1]
```diff
@@ -57,6 +57,7 @@ subpackages="
 	cargo-zsh-completion:_cargo_zshcomp:noarch
 	cargo-doc:_cargo_doc:noarch
 	"
+# TODO: remove cargo-tests-fix-build-target.patch if https://github.com/rust-lang/cargo/pull/5859 be merged to cargo
```
Comment (jirutka) [inline community/rust/APKBUILD]: Thank you for sending patch to upstream!

**Comment by jirutka** [comment]
I've started working on it before you've opened this PR and didn't checked PRs queue before merging; sorry about that.

**Comment by jeizsm** [comment]
No problem

**Pull request closed without merging**